feat: add --auto-cancel-after-failures flag (#25237)
cypress-io/cypress: cli/CHANGELOG.md

**Features:**

- Added the "Open in IDE" feature for failed tests reported from the Debug page. Addressed in [#25691](https://github.com/cypress-io/cypress/pull/25691).

> - Added "--auto-cancel-after-failures" flag that would override the "Auto Cancellation" feature when recording to the cloud. Addressed in [#25237](https://github.com/cypress-io/cypress/pull/25237)

Reviewer: ```suggestion
- Added "--auto-cancel-after-failures" flag that would override the "Auto Cancellation" feature when recording to the Cloud. Addressed in [#25237](https://github.com/cypress-io/cypress/pull/25237)
```

> - Added "--auto-cancel-after-failures" flag that would override the "Auto Cancellation" feature when recording to the Cloud. Addressed in [#25237](https://github.com/cypress-io/cypress/pull/25237)

Reviewer 2: ```suggestion
- Added a new CLI flag, called [`--auto-cancel-after-failures`](https://docs.cypress.io/guides/guides/command-line#Options), that overrides the project-level CI ["Auto Cancellation"](https://docs.cypress.io/guides/cloud/smart-orchestration#Auto-Cancellation) value when recording to the Cloud. This gives Cloud users on Business and Enterprise plans the flexibility to alter the auto-cancellation value per run. Addressed in [#25237](https://github.com/cypress-io/cypress/pull/25237)
```

> - Added a new CLI flag, called [`--auto-cancel-after-failures`](https://docs.cypress.io/guides/guides/command-line#Options), that overrides the project-level CI ["Auto Cancellation"](https://docs.cypress.io/guides/cloud/smart-orchestration#Auto-Cancellation) value when recording to the Cloud. This gives Cloud users on Business and Enterprise plans the flexibility to alter the auto-cancellation value per run. Addressed in [#25237](https://github.com/cypress-io/cypress/pull/25237)

Reviewer 2: ```suggestion
- Added a new CLI flag, called [`--auto-cancel-after-failures`](https://docs.cypress.io/guides/guides/command-line#Options), that overrides the project-level CI ["Auto Cancellation"](https://docs.cypress.io/guides/cloud/smart-orchestration#Auto-Cancellation) value when recording to the Cloud. This gives Cloud users on Business and Enterprise plans the flexibility to alter the auto-cancellation value per run. Addressed in [#25237](https://github.com/cypress-io/cypress/pull/25237).
```
